### DISCUSSION TITLE
Removing Changelogs from main readme.

### DISCUSSION THREAD

**Discussion by theetherGit** [etherCorps/sveltekit-og#21] [Ideas]
@jasongitmail What do you say about this. I want to keep readme clean and straightforward.

We can have a changelog file whch will make it easier to find things.

I'm saying because i saw changelogs in readme and as of now it stands at 1.2.3 and which asks them to install dependency. I think we Should remove it from main readme.

Can i ask for your twitter handle?

**Comment by jasongitmail** [comment]
Seems like a good idea to move it into `CHANGELOG.md`. I believe that's the filename that's standardized.
Could also consider using this standardized format: https://keepachangelog.com/en/1.0.0/ 

[https://twitter.com/zkjason_](https://twitter.com/zkjason_)

**Comment by theetherGit** [comment]
I'm also thinking of standardizing this library. I'll be working on this today. So if you have anything to suggest any changes please mention them here. I'll be completing them by today.

**Comment by jasongitmail** [comment]
Adding a playwright test for each method would be good. Otherwise, looks good with SvelteKit's defaults (prettier, eslint, etc).

**Reply by theetherGit** [reply]
I thought of that too but i didn't had that much time for test. If you have sometime can you do that? 

For now i'm good with Sveltekit views.

**Reply by jasongitmail** [reply]
No time currently. Maybe later.

**Reply by theetherGit** [reply]
It's ok.

**Comment by jasongitmail** [comment]
Another idea for standardizing the library: consider switching to NPM. If you're opposed to it, that's ok. But it would align with SvelteKit more since they updated their docs to change all `pnpm` references to `npm`.

**Reply by theetherGit** [reply]
I'm not opposing it. I'm use to `pnpm` for my personal that's all on actions and other we can go with `npm`.

**Comment by theetherGit** [comment]
@jasongitmail I found a middle ground for build test so we can keep an eye on build errors for any runtime. I don't think so `big test `are required as of now.

Auto Publish is working

Changelog is left.

**Comment by theetherGit** [comment]
Need to work on changelog and docs
